cli/src/semgrep/engine.py

```diff
@@ -59,6 +63,12 @@ def decide_engine_type(
             ):
                 requested_engine = cls.PRO_INTRAFILE
 
+        if requested_engine != cls.OSS and supply_chain_only:
+            logger.info(
+                "Running only supply chain rules so running without Semgrep Pro Engine"
+            )
+            return cls.OSS
```

Comment:
Might as well use `PRO_LANG` instead (or `PRO_INTRAFILE`)

Reply:
Oh instead of OSS?